platform/machine/unprivqemu/cluster.go

```diff
@@ -126,16 +129,19 @@ func (qc *Cluster) NewMachineWithOptions(userdata *conf.UserData, options platfo
 
 	// Default to user mode networking
 	qmCmd = append(qmCmd, "-netdev", "user,id=eth0,hostfwd=tcp:127.0.0.1:0-:22", "-device", platform.Virtio(qc.flight.opts.Board, "net", "netdev=eth0"))
+
 	// Bind the TPM device
-	qmCmd = append(qmCmd, "-chardev", fmt.Sprintf("socket,id=chrtpm,path=%s", swtpmSock), "-tpmdev", "emulator,id=tpm0,chardev=chrtpm", "-device", "tpm-tis,tpmdev=tpm0")
+	if qc.flight.opts.NoSwtpm == false {
+		qmCmd = append(qmCmd, "-chardev", fmt.Sprintf("socket,id=chrtpm,path=%s", swtpmSock), "-tpmdev", "emulator,id=tpm0,chardev=chrtpm", "-device", "tpm-tis,tpmdev=tpm0")
```

Comment:
wanted to pull socket creation here too..but when i did that i was getting an error like this:
`qemu-system-x86_64: -chardev socket,id=chrtpm,path=/tmp/kola-swtpm887381870/swtpm-sock: Failed to connect socket /tmp/kola-swtpm887381870/swtpm-sock: No such file or directory`

Not sure if it is a timing issue in creating the socket